CHANGELOG.md

```diff
@@ -0,0 +1,506 @@
+- added bounding box tests
```

Comment:
You don't need to add this file. To add a changelog entry for this PR follow the instructions here: https://github.com/DKISTDC/dkist/tree/main/changelog